client/components/cart/cart.scss

```diff
@@ -0,0 +1,11 @@
+.deleteButton {
+  background: none!important;
+  border: none;
+  padding: 0!important;
+  /*optional*/
+  font-family: arial, sans-serif;
+  /*input has OS specific font-family*/
+  color: #069;
+  text-decoration: underline;
+  cursor: pointer;
+}
```

Comment:
:'( u don't like bootstrap?

Reply:
I dont know bootstrap that well... u r welcome to change it :)